mne/commands/tests/test_commands.py

```diff
@@ -202,4 +203,37 @@ def test_watershed_bem():
         mne_watershed_bem.run()
 
 
+@slow_test
+@requires_mne
+@requires_freesurfer
+@sample.requires_sample_data
+def test_flash_bem():
```

Comment:
@dgwakeman yeah I source `SetUpFreeSurfer.sh` in my `.bashrc`. I don't know of anything weird in my Python settings that should break anything.